common/circleCI/index.js

```diff
@@ -0,0 +1,14 @@
+import CircleCI from 'circleci'
+
+let circleCI
+let circleCIInitialised = false
+
+if(!circleCIInitialised){
+  circleCIInitialised = true
+  circleCI = new CircleCI({
+    auth: process.env.CIRCLE_TOKEN
+  })
+}
+
+export default circleCI
```

Comment:
my understanding of the way imports work in js means you don't need the `if` at all? 
```
export default new CircleCI({
    auth: process.env.CIRCLE_TOKEN
  })
```

each file gets run once (the first import the code comes across), subsequent imports don't rerun the code they just get the previous result ;)